test/unit/plugins/rest_spec.js

```diff
@@ -51,18 +53,22 @@ describe("The Rest plugin", function () {
 			});
 		});
 	});
-
```

Comment:
I think the extra line of whitespace here is a good thing (just my opinion). Was it removed intentionally?